src/raftstore/store/config.rs

```diff
@@ -177,7 +177,7 @@ impl Default for Config {
             notify_capacity: 40960,
             snap_mgr_gc_tick_interval: ReadableDuration::minutes(1),
             snap_gc_timeout: ReadableDuration::hours(4),
-            messages_per_tick: 4096,
+            messages_per_tick: 128,
```

Comment:
may using 128 reduce the batch size and cause more IOs?

Reply:
In one benchmark case, the write operations of raft engine grows from 220K to 250K. So let's keep the original value, and use a metrics to monitor the message batch size.